src/secp256k1/contrib/lax_der_parsing.c

```diff
@@ -0,0 +1,150 @@
+/**********************************************************************
```

Comment:
Since this file duplicates one that we changed, I suggest deleting it (and lax_der_parsing.h).